### PR TITLE
feat: corrected geometry, out-of-box 3D geom visualization works

### PR DESCRIPTION
**Pull request by TfedUD**
With lower_left_verts now in ladybug_geometry:
   Geometry is functioning properly, full 3D viz 4.0.
   Geom intersection 4.0.

### CODE REVIEW

**Comment by TfedUD** [comment]
I'm 95% sure the fails are due to I manually upgraded the version of all the lbt-core installs and manually installed git+most recent lbt-geometry

**Comment by TfedUD** [comment]
Woohoo fixed it!

File: dragonfly_doe2/doe/hvac.py [hunk 1]
```diff
@@ -13,3 +16,11 @@ def to_inp(self):
 
     def __repr__(self):
         return self.to_inp()
+
+
+@dataclass
+class _HVACBase(object):
```
Comment (mostaphaRoudsari) [inline dragonfly_doe2/doe/hvac.py:22]: You don't need `object` in Python 3.

```suggestion
class _HVACBase:
```

File: makedoe.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import pathlib
```
Comment (mostaphaRoudsari) [inline makedoe.py:1]: I imagine you are using this file for testing - let's remove it from this PR.

**Review by mostaphaRoudsari — APPROVED** [review]
Hi @TfedUD! I left two minor comments. It LGTM!

**Pull request merged**